core/templates/components/forms/custom-forms-directives/image-uploader.directive.ts

```diff
@@ -36,16 +37,17 @@ angular.module('oppia').directive('imageUploader', [
         onFileChanged: '=',
         errorMessage: '@',
         width: '@',
-        getAllowedImageFormats: '&allowedImageFormats'
+        getAllowedImageFormats: '&allowedImageFormats',
+        backgroundWhileUploading: '@'
       },
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/components/forms/custom-forms-directives/' +
         'image-uploader.directive.html'),
       link: function(scope: ImageUploaderCustomScope, elt) {
         var onDragEnd = function(e) {
           e.preventDefault();
-          $('.image-uploader-drop-area').removeClass(
-            'image-uploader-is-active');
+          scope.backgroundWhileUploading = false;
+          scope.$apply();
```

Comment:
Just to check, does this not work without the $apply()? ditto below.

Reply:
It's not working without $apply()

Reply:
Do we know why?

Reply:
We are using jqlite event callbacks and I guess they run "outside" angularJS, but I didn't find it clearly stated in the docs.

Reply:
Ah okay, makes sense. Can you please add a comment for this in the code so that developers know why it is required and they don't accidentally remove it (do this in both places in this file)? Thanks!

Reply:
sure! just a min

Reply:
> Ah okay, makes sense. Can you please add a comment for this in the code so that developers know why it is required and they don't accidentally remove it (do this in both places in this file)? Thanks!

done!